internal/controllers/machine_controller.go

```diff
@@ -221,7 +221,12 @@ func (r *MachineReconciler) Start(ctx context.Context) error {
 }
 
 func (r *MachineReconciler) startCheckAndEnqueueVolumeResize(ctx context.Context, log logr.Logger) {
+	if r.resyncIntervalVolumeSize == 0 {
+		return
```

Comment:
Should we print an info if the volume resize is disabled?

Reply:
sure we can.

Reply:
maybe not, this technically doesn't disable resize logic. It will just disable triggering of resize base on volume size. If this volume size check loop is deactivated, it doesn't mean resize is disabled. Resize will still executed in reconcile loop. Currently i think all machines are reconcile every hour, if you don't have any other trigger mechanism of reconcile loop.
So just simple message about volume resize is disabled will be more confusing for users because it isn't true.
Technically we can add logs: Reconciliation of volume size base on change of capacity (or size) is disabled.

But this won't be very useful for users too without context and we don't have any documentation where we describe how libvirt-provider is working.

Reply:
updated @lukasfrank